routes.rb

```diff
@@ -87,7 +95,16 @@ class App < Sinatra::Application
     return {:success => false, :message => I18n.t(:enter_email)}.to_json unless valid_email?(params[:email])
     mandrill = Mandrill.new(session[:key])
     return {:can_connect => false, :message => I18n.t(:mandrill_cannot_connect), :goto_section => 'connect_mandrill'}.to_json unless mandrill.can_connect?
-    response = mandrill.send_single_email(session[:template], params[:email], [{name: 'FULLNAMEORUSERNAME', content: 'King Kong'}])
+
+    begin
+      command = Database.connection.create_command(session[:db_query])
```

Comment:
not sure if executing the query again is the best idea? Should we cache the query result IF it was executed in the previous step?